### ISSUE TITLE
Error piecewise when simulating fish spatial example

### ISSUE THREAD

**Issue by matthiaskoenig** [spatial-model-editor/spatial-model-editor#816]
I just tried the 
https://github.com/sbmlteam/sbml-spatial-models/blob/master/Analytic%20models/fish_300x300.xml
example.
The support for spatial looks great. I especially like the geometry part with the compartments.

When trying to run the simulation I get an error about piecewise (see below)

```
Simulation setup failed.

Error message: IOError [handle_parser_error:/opt/smelibs/include/dune/copasi/common/pdelab_expression_adapter.hh:35]: Evaluating muParser expression failed:
  Parsed expression:   -1.0*(-0.02*u + piecewise(0.5, 0.5 <= 0.37 - 0.04*v - 0.055*w, 0, true) + piecewise(0.37 - 0.04*v - 0.055*w, and(0 < 0.37 - 0.04*v - 0.055*w, 0.37 - 0.04*v - 0.055*w < 0.5), 0, true)) - 1.0*(-0.025*v + piecewise(0.5, 0.5 <= 0.25 - 0.05*u, 0, true) + piecewise(0.25 - 0.05*u, and(0 < 0.25 - 0.05*u, 0.25 - 0.05*u < 0.5), 0, true)) - 1.0*(-0.06*w + piecewise(0.5, 0.5 <= 0.24 + 0.016*u - 0.03*v, 0, true) + piecewise(0.24 + 0.016*u - 0.03*v, and(0 < 0.24 + 0.016*u - 0.03*v, 0.24 + 0.016*u - 0.03*v < 0.5), 0, true)) 
  Token:               piecewise
  Error position:      16
  Error code:          16
  Error message:       Unexpected token "piecewise" found at position 16.
```

Using the compiled linux version on Ubuntu 20.04.
Best M

**Comment by matthiaskoenig** [comment]
Sorry, just read your piecewise comment. Just ignore this issue.

**Comment by lkeegan** [comment]
Thanks for the report - the latest 1.3.1 release available to download contains #815 so I'll close this issue.

**Comment by matthiaskoenig** [comment]
Thanks.
